src/symbol/glyph_source.js

```diff
@@ -50,22 +55,33 @@ class GlyphSource {
         // the number of pixels the sdf bitmaps are padded by
         const buffer = 3;
 
-        const missing = {};
+        const missingRanges = {};
         let remaining = 0;
 
         const getGlyph = (glyphID) => {
             const range = Math.floor(glyphID / 256);
+            if (this.localIdeographFontFamily &&
+                (isChar['CJK Unified Ideographs'](glyphID) ||
+                 isChar['Hangul Syllables'](glyphID))) {
```

Comment:
That's an interesting question. On the JS side, I suspect it would mostly just work to have `> maxuint16` glyph IDs. My bigger concern would be about glyph shape (especially glyph advances). Even within the BMP I've been conservative with what ranges to include -- looking for the biggest performance boost with the least risk of introducing rendering errors. Do you have a particular range you think would be a good candidate for trying out?

If we implement a FreeType analogue of this on the iOS/Android side, we'd be able to extract glyph metrics directly from local fonts, so there'd be a much more direct path to supporting the supplementary planes (note: I don't actually know how to get a "system font" in Android, but I think it should be possible).